src/pages/NewAccount/Confirmation/Confirmation.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <mobile-page
+    :redirect-to-on-close="{ name: 'new-account-intro' }"
+    class="confirmation"
+    title="New Account"
+    back-button
+  >
+    <h1>
+      Confirm your phrase
+    </h1>
+    <div class="mnemonic">
+      <p v-if="tapedWords.length === 0">
+        Tap the words below to
+        add them here, note the
+        correct order!
+      </p>
+      <p
+        v-show="tapedWords"
+        class="phrase"
+      >
+        <span
+          v-for="(word, index) in tapedWords"
+          :key="`${word}-${index}`"
+        >
+          {{ word }}
+        </span>
+      </p>
+    </div>
+    <p class="icon-hand">
+      👆
+    </p>
+    <div class="words">
+      <span
+        v-for="(word, index) in words"
+        :key="`${word}-${index}`"
+        class="word"
+      >
+        <ae-button
+          size="small"
+          type="exciting"
+          uppercase
+          @click="tapWord($event, word)"
+        >
+          {{ word }}
+        </ae-button>
+      </span>
+    </div>
+    <ae-modal-light
+      v-if="validPhrase"
+      title="Correct Passphrase!"
+    >
+      Proceed to your æpp browser. Enjoy
+      the æpp ecosystem!
+      <ae-divider/>
+      <ae-button
+        slot="buttons"
+        size="smaller"
+        uppercase
+        @click="setSeed"
+      >
+        CREATE PASSWORD
+      </ae-button>
+    </ae-modal-light>
+    <ae-modal-light
```

Comment:
Becomes reasonable to extract AeAlertModal or AeConfirmModal component with a single button to reduce code duplication.